src/annotation/index.ts

```diff
@@ -6,6 +6,25 @@ import {
   AuditAction,
 } from "./interfaces";
 
+/* eslint-disable */
+function log(
+  target: any,
```

Comment:
I ?think? this is `target: typeof Annotations`. Could also be `never` as we don't use it?

Reply:
From what I read, target is the class that the decorated method belongs to so yeah, it should be `Annotations`.